web/src/hooks/UseOrganizations.ts

```diff
@@ -38,8 +38,8 @@ export function useOrganizations() {
     ['organization', 'superuser', 'organizations'],
     fetchOrgsAsSuperUser,
     {
-      enabled: isSuperUser,
       retry: false,
+      enabled: isSuperUser === true && !loading,
```

Comment:
prevents 401 errors on superuser APIs during Organization list page re-renders since `isSuperuser` was `undefined` since the api query didn't complete on the 1st render